RELEASING.md

```diff
@@ -1,9 +1,26 @@
-# Releasing
+# How to release v1
 
-1. Make changes.
-2. Determine next version. For the sake of this list, let's assume 1.2.3.
-3. Update `gax.go` with `const Version = "1.2.3"`. If v2, do the same to `v2/gax.go`.
-4. `git checkout -b my_branch && git commit && git push my_remote my_branch`
-5. Get it reviewed and merged.
-6. `git checkout master && git tag v1.2.3 && git push origin v1.2.3`
-7. Update the [releases page on GitHub](https://github.com/googleapis/gax-go/releases)
+1. Determine the current release version with `git tag -l`. It should look
+   something like `vX.Y.Z`. We'll call the current version `$CV` and the new
+   version `$NV`.
+1. On master, run `git log $CV..` to list all the changes since the last
+   release.
+   a. NOTE: Some commits may pertain to only v1 or v2. Manually introspect
+   each commit to figure which occurred in v1.
+1. Edit `CHANGES.md` to include a summary of the changes.
+1. Mail the CL containing the `CHANGES.md` changes. When the CL is approved,
+   submit it.
+1. Without submitting any other CLs:
+   a. Switch to master.
+   b. `git pull`
+   c. Tag the repo with the next version: `git tag $NV`. It should be of the
+   form `v1.Y.Z`.
+   d. Push the tag: `git push origin $NV`.
+1. Update [the releases page](https://github.com/googleapis/google-cloud-go/releases)
+   with the new release, copying the contents of the CHANGES.md.
+
+# How to release v2
+
+Same process as v1, once again noting that the commit list may include v1
+commits (which should be pruned out). Note also whilst v1 tags are `v1.Y.Z`, v2
+tags are `v2.Y.Z`.
```

Comment:
Also have to update the `go.mod` with `/v2` and all local imports to use `/v2`.

Reply:
That's a one time change (when initially creating the go.mod), no? As in, you don't need to muck with the go.mod or imports each time you releasing v2 I think?

Reply:
Right. It kind of depends how the repo is structured. Don't want to accidentally forget. Could just be something like:

> Note: the first time you upgrade to a new version, you must update the module path in `go.mod` to include `/vN` and all local imports to use the new `/vN` path.

Reply:
Ahhh gotcha. I've added a link to the appropriate wiki section. Does that suffice?